apiary.apib

```diff
@@ -1997,6 +1999,7 @@ Get a quotation
     + Attributes (object)
         + data (object)
             + id: `e7a3fe2b-2c75-480f-87b9-121816b5257b` (string)
+            + name: `Quotation 1234` (string)
```

Comment:
Q: For deals I see we call this a `title`, would it make sense to keep the same naming here too?

Reply:
Good remark. I vote for consistency here.

Reply:
I will change it to title.